crates/common/tedge_config/src/tedge_toml/tedge_config_location.rs

```diff
@@ -1,16 +1,14 @@
 use std::path::Path;
 
-use crate::tedge_toml::figment::ConfigSources;
-use crate::tedge_toml::figment::FileAndEnvironment;
-use crate::tedge_toml::figment::FileOnly;
-use crate::tedge_toml::figment::UnusedValueWarnings;
+use crate::tedge_toml::DtoKey;
```

Comment:
I don't get where this `DtoKey` is defined.

Reply:
It's defined along with `WritableKey` etc in [query.rs](https://github.com/thin-edge/thin-edge.io/pull/3518/files#diff-cd95e9aea2dc9e842e4cd738eaf06c1e284ccab0754a684defe0851b188a7faeR53)